proposals/PlayServicesVersion_Proposal.md

```diff
@@ -0,0 +1,62 @@
+# Play Services Version Proposal
+
+- Status: Proposed
+
+## Overview
+This is a proposal to offer an alternative solution for [CB-13145](https://issues.apache.org/jira/browse/CB-13145).
+
+## Problem
+Plugins using different versions of Google Play Services cause gradle to error out with the following error:
```

Comment:
Plugins using different versions of Google Play Services or the `+` to indicate range cause gradle to error out with the following error: